src/main/java/com/aws/greengrass/deployment/IotJobsHelper.java

```diff
@@ -361,7 +361,6 @@ private void setupCommWithIotJobs() {
                     logger.atWarn().log("Interrupted while subscribing to Iot Jobs topics");
                     return;
                 }
-                logger.atInfo().log("Connection established to IoT cloud");
```

Comment:
question: do we log somewhere when we are connected? In the MQTT client itself?

Reply:
Yeah, in the MQTT client.